src/PSRule.Rules.Azure/rules/Azure.SQL.Rule.ps1

```diff
@@ -94,7 +94,7 @@ Rule 'Azure.SQL.TDE' -Ref 'AZR-000191' -Type 'Microsoft.Sql/servers/databases' -
         return $Assert.Fail($LocalizedData.SubResourceNotFound, 'Microsoft.Sql/servers/databases/transparentDataEncryption');
     }
     foreach ($config in $configs) {
-        $Assert.HasFieldValue($config, 'Properties.status', 'Enabled');
+        $Assert.HasFieldValue($config, 'Properties.state', 'Enabled');
```

Comment:
The property `apiVersion` will not exist if exported with _in-flight_ scenarios, but latest API version should be used. We only need to catch the case when older IaC is used.

```suggestion
        if ($Assert.HasFieldValue($config, 'apiVersion', '2014-04-01').Result) {
            $Assert.HasFieldValue($config, 'Properties.status', 'Enabled');
        }
        else {
            $Assert.HasFieldValue($config, 'Properties.state', 'Enabled');
        }
```